src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -40,15 +41,22 @@ def connect_config_db_for_ns(namespace=DEFAULT_NAMESPACE):
       handle to the config_db for a namespace
     """
     SonicDBConfig.load_sonic_global_db_config()
-    config_db = ConfigDBConnector(namespace=namespace)
+    if namespace is not None and namespace != DEFAULT_NAMESPACE:
+        config_db = ConfigDBConnector(use_unix_socket_path=True, namespace=namespace)
+    else:
+        config_db = ConfigDBConnector(namespace=namespace)
```

Comment:
why do we need this change ?

Reply:
These changes are not required for this fix. I'll remove.

Reply:
Fixed.